iron-validatable-behavior.html

```diff
@@ -16,6 +16,40 @@
   /**
    * Use `Polymer.IronValidatableBehavior` to implement an element that validates user input.
    *
+   * This element can be used in two different ways:
+   *   1. to add a `validate()` method to a custom element that you are implementing,
+   * thus making it validatable
+   *   2. to add custom validation logic to an existing `<iron-input>`.
```

Comment:
So, when we get down to #2, you're actually describing how to extend `IronValidatorBehavior`, so it's a little confusing that you don't mention the two behaviors up here. I wonder if it might make sense to mention both up front, like: 

Use `Polymer.IronValidatableBehavior` to  implement an element that validates user input.

Use the related `Polymer.IronValidatorBehavior` to add custom validation logic to an `iron-input`. 

(Is it worth mentioning that you can actually use a custom validator with anythign that mixes in Validatable, as long as they haven't overridden _getValidity?)